icons/samsungpay.svg

```diff
@@ -0,0 +1 @@
+<svg aria-labelledby="simpleicons-samsungpay-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-samsungpay-icon">Samsung Pay icon</title><path d="M7.932.133C6.587.37 4.8 1.025 3.83 1.63 2.179 2.673 1.14 4.305.39 7.013.034 8.308 0 8.795 0 12.226c0 4.222.12 4.962 1.174 7.182.817 1.682 1.737 2.607 3.405 3.415C6.79 23.899 7.455 24 12 24s5.209-.1 7.421-1.177c1.668-.808 2.588-1.733 3.405-3.415C23.88 17.188 24 16.448 24 12.226c0-3.43-.034-3.918-.392-5.213C22.553 3.178 20.8 1.378 17.192.369 15.983.033 9.464-.136 7.932.133z" fill="#000"/><path d="M13.234 9.12v.38c-.39-.32-.878-.5-1.387-.5-1.258 0-2.286 1.098-2.286 2.446 0 1.347 1.018 2.445 2.286 2.445.509 0 .998-.18 1.387-.499v.42h1.048V9.12h-1.048zm-1.268 3.853c-.758 0-1.377-.679-1.377-1.527 0-.849.619-1.518 1.377-1.518.759 0 1.378.68 1.378 1.528s-.62 1.517-1.378 1.517zM6.686 9c-.51 0-1.009.18-1.398.52v-.4H4.23v6.398h1.048v-2.136c.39.33.889.51 1.398.52 1.257 0 2.286-1.099 2.286-2.446C8.962 10.108 7.953 9 6.686 9zm-.12 3.973c-.759 0-1.378-.679-1.378-1.527 0-.849.62-1.528 1.378-1.528.758 0 1.377.68 1.377 1.528s-.619 1.527-1.377 1.527zM18.744 9.12l-1.347 3.194-1.438-3.194h-1.038l1.977 4.392-.849 2.006h1.028l2.695-6.398h-1.028z" fill="#fff/></svg>
```

Comment:
Seems like you have two `fill`'s in there - could you remove those (probably by knocking out the white one from the square)?